examples-copier/configs/config.example.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "source_directory": "generated-examples/go",
```

Comment:
I noticed the introduction of a `COPIER_COMMIT_STRATEGY` in the `github_write_to_target.go` file, but I don't see that being set anywhere. Is this something we could set as part of this configuration? i.e. "when writing to `example-repo` copy directly to `main`, but when writing to `another-repo` make a PR instead"?

Reply:
added as a config.json value, and updated `github_write_to_target.go` to read it to determine the strategy (pr = default)